src/cmd/run.go

```diff
@@ -51,6 +51,10 @@ var runCmd = &cobra.Command{
 	Run: func(_ *cobra.Command, args []string) {
 		var tasksFile types.TasksFile
 
+		if _, err := os.Stat(config.TaskFileLocation); os.IsNotExist(err) {
+			message.Fatalf(err, "file does not exist: %s", config.TaskFileLocation)
+		}
+
 		err := utils.ReadYaml(config.TaskFileLocation, &tasksFile)
```

Comment:
technically we just shouldn't rewrite the error coming out of this function - it should return the right error (which could be permissions too) we just need to print it instead of the custom message

Reply:
@Racer159 Can you take a look again? I handled it so the rewriting of the messages are only to make the wording sound consistent and so ReadYaml properly outputs either the file access errors or the "cannot unmarshal" with an abbreviated reason why.